src/Cropperjs/.gitignore

```diff
@@ -1,4 +1,5 @@
-vendor
-composer.lock
-.php_cs.cache
-.phpunit.result.cache
+/assets/node_modules/
```

Comment:
Should be in your local GIT config
```suggestion
```

Reply:
Mmmh not really, some projects commit their `node_modules` folder,  plus I don't really think there is a lot of people that globally git ignore this folder

Reply:
Well let's add it only when necessary then 😅